app/blueprints/round/templates/round_details.html

```diff
@@ -0,0 +1,544 @@
+{% from "govuk_frontend_jinja/macros/attributes.html" import govukAttributes -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% from "govuk_frontend_jinja/components/table/macro.html" import govukTable %}
+{%- from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary -%}
+{%- from "govuk_frontend_jinja/components/details/macro.html" import govukDetails -%}
+{% from "govuk_frontend_jinja/components/notification-banner/macro.html" import govukNotificationBanner %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{%- from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList -%}
+{% set showNavigationBar = True %}
+{% set active_item_identifier = "applications" %}
+{% extends "base.html" %}
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        {{ govukBackLink({
+          "text": "Back",
+          "href": url_for("round_bp.view_all_rounds")
+        }) }}
+    </div>
+</div>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+    {% with messages = get_flashed_messages() %}
+        {% if messages %}
+            {% set html %}
+                <p class="govuk-notification-banner__heading">{{ messages[0] }}</p>
+            {% endset %}
+            <div class="govuk-grid-row govuk-notification-banner__heading">
+                {{ govukNotificationBanner({"html": html,"type": "success"}) }}
+            </div>
+        {% endif %}
+    {% endwith %}
+    <div class="govuk-grid-column-full">
+</div>
+<div class="govuk-grid-row">
+   <div class="govuk-grid-column-full">
+    <div class="govuk-grid-column-three-quarters govuk-!-padding-0">
+        <h2 class="govuk-heading-l govuk-!-margin-bottom-0">Apply for {{ round.fund.title_json["en"] }}</h2>
+        <a class="govuk-link govuk-!-margin-top-0 govuk-link--no-visited-state" href="{{ url_for("fund_bp.edit_fund", fund_id=round.fund.fund_id) }}">Change application heading</a>
```

Comment:
Please can this link be anchored to the specific form field the user should edit (grant_title/application_heading)?

Reply:
@samuelhwilliams Thank for spotting this.. fix pushed